templates/patroni.yml.j2

```diff
@@ -100,7 +100,11 @@ bootstrap:
   {%- if restoring_backup %}
   method: pgbackrest
   pgbackrest:
+    {%- if restoring_to_pitr_target %}
+    command: pgbackrest {{ pgbackrest_configuration_file }} --stanza={{ restore_stanza }} --pg1-path={{ data_path }} --set={{ backup_id }} --target="{{ pitr_target }}" --type=time --target-action=promote restore
```

Comment:
@Zvirovyi, do you have a Jira ticket with the accepting criteria for this implementation? I'm curious about the details, especially if it's okay to allow PITR without providing the backup ID.

Reply:
Added Jira ticket and will add instruction